service/src/main/kotlin/app/cash/backfila/service/BackfilaDevelopmentService.kt

```diff
@@ -122,3 +132,30 @@ class DevelopmentViewLogsUrlProvider : ViewLogsUrlProvider {
     return "/"
   }
 }
+
+class SelfBackfillingModule : KAbstractModule() {
```

Comment:
this is what we'll need to add in our service modules

Reply:
Should it be in a separate config file?